### PR TITLE
JAR-5164

### PR DESCRIPTION
**Pull request by nimbixler**

### CODE REVIEW

**Review by kenhill — APPROVED** [review]

File: InfiniBandCNI.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# CNI Support for MPI over InfiniBand
+
+The following document is a HOWTO for how to enable Kubernetes support for using MPI over InfiniBand in JARVICE jobs.  Note that the referrenced components are managed by their respective third party maintainers.
+
+## Howto enable IPoIB in Kubernetes.
```
Comment (oxedions) [inline InfiniBandCNI.md:5]:
```suggestion
## Howto enable IPoIB in Kubernetes
```

File: InfiniBandCNI.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Assumptions
+- working Kubernetes cluster with a functional primary CNI
+- required drivers already installed for infiniband host adapter
+- an existing IPoIB interface is up and active on the hosts
+- This has only been tested with Mellanox infiniband cards and Mellanox OFED
+
+### Required 3rd party tools
+- [Multus](https://github.com/k8snetworkplumbingwg/multus-cni) project will be utilized to add the secondary IpoIB interface. 
+ - [ipoib-cni](https://github.com/Mellanox/ipoib-cni) from Mellanox
+ - [Whereabouts](https://github.com/dougbtv/whereabouts) for ipam
```
Comment (oxedions) [inline InfiniBandCNI.md:16]:
```suggestion
- [Multus](https://github.com/k8snetworkplumbingwg/multus-cni) project will be utilized to add the secondary IpoIB interface
- [ipoib-cni](https://github.com/Mellanox/ipoib-cni) from Mellanox
- [Whereabouts](https://github.com/dougbtv/whereabouts) for ipam
```

File: InfiniBandCNI.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+1. Install Multus by applying [multus-daemonset-thick-plugin.yaml](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/deployments/multus-daemonset-thick-plugin.yml). 
+```
+kubectl apply -f multus-daemonset-thick-plugin.yml
+```
+2. deploy the [ipoib-cni](https://github.com/Mellanox/ipoib-cni/blob/master/images/ipoib-cni-daemonset.yaml)
```
Comment (oxedions) [inline InfiniBandCNI.md:22]:
```suggestion
2. Deploy the [ipoib-cni](https://github.com/Mellanox/ipoib-cni/blob/master/images/ipoib-cni-daemonset.yaml).
```

File: InfiniBandCNI.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+The following document is a HOWTO for how to enable Kubernetes support for using MPI over InfiniBand in JARVICE jobs.  Note that the referrenced components are managed by their respective third party maintainers.
+
+## Howto enable IPoIB in Kubernetes.
+
+Assumptions
+- working Kubernetes cluster with a functional primary CNI
+- required drivers already installed for infiniband host adapter
+- an existing IPoIB interface is up and active on the hosts
+- This has only been tested with Mellanox infiniband cards and Mellanox OFED
```
Comment (oxedions) [inline InfiniBandCNI.md:11]:
```suggestion
- this has only been tested with Mellanox infiniband cards and Mellanox OFED
```

File: InfiniBandCNI.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+1. Install Multus by applying [multus-daemonset-thick-plugin.yaml](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/deployments/multus-daemonset-thick-plugin.yml). 
+```
+kubectl apply -f multus-daemonset-thick-plugin.yml
+```
+2. deploy the [ipoib-cni](https://github.com/Mellanox/ipoib-cni/blob/master/images/ipoib-cni-daemonset.yaml)
+```
+kubectl apply -f ipoib-cni-daemonset.yaml
+```
+3. Install the whereabouts ipam
```
Comment (oxedions) [inline InfiniBandCNI.md:26]:
```suggestion
3. Install the whereabouts ipam.
```

File: values.yaml [hunk 6]
```diff
@@ -165,6 +165,11 @@ jarvice:
   JARVICE_S3_PREFIX: "output/" # object name prefix
   JARVICE_S3_ENDPOINTURL: # object store endpoint URL, defaults to AWS
 
+  # the following value should be set to the value needed for the annotation
+  # k8s.v1.cni.cncf.io/networks to specify the Multus CNI name for passing in
+  # IPoIB networks (needed when using the ibrdma plugin)
+  JARVICE_IB_CNI_NETWORKS:  # defaults to jarvice-ipoib
```
Comment (oxedions) [inline values.yaml:171]:
```suggestion
  JARVICE_IB_CNI_NETWORKS: # defaults to jarvice-ipoib
```

File: InfiniBandCNI.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
+# CNI Support for MPI over InfiniBand
+
+The following document is a HOWTO for how to enable Kubernetes support for using MPI over InfiniBand in JARVICE jobs.  Note that the referrenced components are managed by their respective third party maintainers.
+
+## Howto enable IPoIB in Kubernetes.
+
+Assumptions
```
Comment (oxedions) [inline InfiniBandCNI.md:7]:
```suggestion
Assumptions:
```

File: InfiniBandCNI.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
+# CNI Support for MPI over InfiniBand
+
+The following document is a HOWTO for how to enable Kubernetes support for using MPI over InfiniBand in JARVICE jobs.  Note that the referrenced components are managed by their respective third party maintainers.
```
Comment (oxedions) [inline InfiniBandCNI.md:3]:
```suggestion
The following document is a HOWTO for how to enable Kubernetes support for using MPI over InfiniBand in JARVICE jobs. Note that the referrenced components are managed by their respective third party maintainers.
```

Comment (oxedions) [reply InfiniBandCNI.md:3]: Note sure, seems recurrent in code, so maybe double space on purpose?

**Review by oxedions — COMMENTED** [review]
Small typo stuff

**Pull request merged**